index.bs

```diff
@@ -480,36 +480,16 @@ starts at 0 the first time {{getCurrentFrameData()}} is invoked for a given
 {{VRDisplay}}.
 
 <dfn attribute for="VRFrameData">leftProjectionMatrix</dfn>
-A 4x4 matrix describing the projection to be used for the left eye's rendering,
-represented as a 16 element array in column major order. This value may be
-passed directly to WebGL's uniformMatrix4fv function. It is highly recommended
-that applications use this matrix without modification. Failure to use this
-projection matrix when rendering may cause the presented frame to be distorted
-or badly aligned, resulting in varying degrees of user discomfort.
+A 4x4 matrix describing the projection to be used for the left eye's rendering, given as a 16 element array in column major order. This value may be passed directly to WebGL's uniformMatrix4fv function. It is highly recommended that applications use this matrix without modification. Failure to use this projection matrix when rendering may cause the presented frame to be distorted or badly aligned, resulting in varying degrees of user discomfort.
```

Comment:
Regardless of how the matrix is stored in memory, there's a difference in convention for how they get multiplied together and with vectors. The historical convention for OpenGL (although  you can do whatever  you want in your shader) is:
newVertex = Matrix * vertex

Whereas for D3D the historical convention is:
newVertex = vertex*Matrix

The former assumes that vectors are columns (and thus can only go on the right in order for the dimensions to make sense), the latter assumes that they're rows and can only go on the left. If we say nothing, people will likely assume the "traditional OpenGL style" (which is what we want), but might be worth calling out to be explicit. E.g. say 

It does have implications for order of multiplications, as  you might expect. E.g.

finalMatrix = projection * view * model; // "OpenGL" convention
finalMatrix = model * view * projection; // "D3D" convention

I call them D3D vs OpenGL convention but you can really go either way depending on how you write your shaders, so there's nothing enforced, just historical convention. The key thing is that the matrices have to be transposed if you switch from one convention to the next. I.e. the x component of the result of the product "vertex*M" is the dot product of vertex and the first *column* of M, whereas for "M*vertex" the x component is the dot product of the first *row* and the vertex, and so on.